Misc/NEWS.d/next/Core and Builtins/2021-07-17-13-41-58.bpo-44662.q22kWR.rst

```diff
@@ -0,0 +1,2 @@
+Fix ``types.Union`` compatibility issue with ``typing.Annotated``.
+Patch provided by Yurii Karabas.
```

Comment:
I just realized this isn't the main addition. The main part is that you added `__name__` etc. to Union, and it coincidentally fixed `typing.Annotated` :).

```suggestion
Add ``__name__``, ``__qualname__`` and ``__module__`` to ``types.Union``.
This also fixes ``types.Union`` issues with ``typing.Annotated``.
Patch provided by Yurii Karabas.
```